content/kotlin/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,390 @@
+
+```kotlin
+fun main(){
+    val a: Float = 3.14f
+    val b: Float = 2.71f
+    val c: Float = a * b as Float
+    print(c)
+
+}
+```
+
+The **output** will be :
+
+```
+8.5094
+```
+In this example, the c variable will be assigned the value 8.5014, which is the result of 3.14 * 2.71.
+
+
+# Int
+
+In Kotlin, the **Int** data type represents a 32-bit signed integer. It is a primitive data type that can hold values ranging from -2147483648 to 2147483647.
+
+Here is an example of how to declare an **Int** variable in Kotlin:
+
+```kotlin 
+fun main(){
+    val a: Int = 50
+    val b: Int = 25
+    val c: Int = a + b
+    print(c)
+}
+```
+
+The **output** of the above code will be :
+```
+75
+```
+
+
+# Long 
+
+In Kotlin, the **Long** data type represents a 64-bit signed integer. It is a primitive data type that can hold values ranging from -9223372036854775808 to 9223372036854775807.
+
+Here is an example of how to declare a **Long** variable in Kotlin:
+
+```kotlin 
+fun main(){
+    val a: Long = 500000000
+    val b: Long = 250000000
+    val c: Long = a + b
+    print(c)
+}
+```
+The **output** for the above code will be :
+```
+750000000
+```
+In this example, the c variable will be assigned the value 750000000, which is the result of 500000000 + 250000000.
+
+
+# Short
+
+In Kotlin, the **Short** data type represents a 16-bit signed integer. It is a primitive data type that can hold values ranging from -32768 to 32767.
+
+Here is an example of how to declare a **Short** variable in Kotlin:
+
+```kotlin
+fun main(){
+    val a: Short = 50
+    val b: Short = 25
+    val c: Short = (a + b).toShort()
+    print(c)
+}
+```
+The **output** for the above code will be:
+```
+75
+```
+In this example, the c variable will be assigned the value 75, which is the result of 50 + 25. The toShort() function is used to explicitly convert the result to a **Short** type.
+
+
+
+## Non-Primitive data type 
+
+We will now discuss the non-primitive data types in kotlin :
+
+-Arrays 
+-String 
+-Any
+-Nothing 
+-Unit 
+
+
+# Arrays
+
+In Kotlin, an **array** is a data structure that stores a fixed-size collection of elements of the same data type. Arrays are non-primitive data types, which means they are derived from primitive data types or other non-primitive data types.
+
+To declare an **array** in Kotlin, you can use the arrayOf() function and specify the elements of the array within the parentheses. For example:
+
+```kotlin 
+fun main(){
+    val numbers: Array<Int> = arrayOf(1, 2, 3, 4, 5)
+}
+```
+
+In this example, the numbers variable is an array of integers that contains the values 1, 2, 3, 4, and 5. The type of the array is specified using the Array<Int> syntax, which indicates that the array contains elements of the Int type.
+
+If you want to know how to print the array then you can check [Arrays](https://github.com/Codecademy/docs/tree/main/content/kotlin/concepts/arrays/arrays.md) to learn further about Arrays.
```

Comment:
This sentence is unnecessary and links to an article that may not be merged before this one.

Reply:
alright i will remove it 👍🏻